.rubocop.yml

```diff
@@ -1,4 +1,5 @@
 require:
+ - rubocop-github-annotation-formatter
```

Comment:
why use this instead of the [builtin --and likely better maintained-- formatter](https://docs.rubocop.org/rubocop/formatters.html#github-actions-formatter)?

Reply:
Ah, ty, I wasn't aware of the builtin one! Fixed.